data/json/items/melee/swords_and_blades.json

```
@@ -3378,5 +3753,203 @@
+    "price": 97000,
+    "price_postapoc": 4000,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "neutral" },
+    "material": [ "hc_steel" ],
+    "repairs_with": [ "hc_steel" ],
+    "techniques": [ "WBLOCK_2", "RAPID", "BRUTAL" ],
+    "qualities": [ [ "CUT", 2 ], [ "BUTCHER", 7 ] ],
+    "flags": [ "SHEATH_SWORD", "CONDUCTIVE" ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS", "FENCING_WEAPONRY" ],
+    "melee_damage": { "bash": 5, "cut": 36 }
+  },
+  {
+    "id": "ch_talwar",
+    "type": "TOOL",
+    "copy-from": "lc_talwar",
+    "name": { "str": "case hardened steel talwar" },
+    "description": "A curved sword used by the Raj of india in the 16th century, a talwar is usually impeccibly decorated and able to take off a limb with a single swipe. \n\nThis one has a proper grip and is made of case hardened steel and is otherwise a fine weapon",
+    "weight": "1425 g",
+    "volume": "1250 ml",
+    "price": 97000,
+    "price_postapoc": 4000,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "neutral" },
+    "material": [ "ch_steel" ],
+    "repairs_with": [ "ch_steel" ],
+    "techniques": [ "WBLOCK_2", "RAPID", "BRUTAL" ],
+    "qualities": [ [ "CUT", 2 ], [ "BUTCHER", 7 ] ],
+    "flags": [ "SHEATH_SWORD", "NONCONDUCTIVE" ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS", "FENCING_WEAPONRY" ],
+    "melee_damage": { "bash": 5, "cut": 37 }
+  },
+  {
+    "id": "qt_talwar",
+    "type": "TOOL",
+    "copy-from": "lc_talwar",
+    "name": { "str": "tempered steel steel talwar" },
+    "description": "A curved sword used by the Raj of india in the 16th century, a talwar is usually impeccibly decorated and able to take off a limb with a single swipe. \n\nThis one has a proper grip and is made of tempered steel, and pre apocalypse a Raj could have used it.",
+    "weight": "1425 g",
+    "volume": "1250 ml",
+    "price": 97000,
+    "price_postapoc": 4000,
+    "to_hit": { "grip": "weapon", "length": "short", "surface": "line", "balance": "good" },
+    "material": [ "qt_steel" ],
+    "repairs_with": [ "qt_steel" ],
+    "techniques": [ "WBLOCK_2", "RAPID", "BRUTAL" ],
+    "qualities": [ [ "CUT", 2 ], [ "BUTCHER", 8 ] ],
+    "flags": [ "SHEATH_SWORD", "NONCONDUCTIVE" ],
+    "weapon_category": [ "MEDIEVAL_SWORDS", "MEDIUM_SWORDS", "FENCING_WEAPONRY" ],
+    "melee_damage": { "bash": 5, "cut": 38 }
+  },
+  {
+    "id": "lc_kilij",
+    "type": "TOOL",
+    "looks_like": "scimitar",
+    "category": "weapons",
+    "symbol": "/",
+    "color": "dark_gray",
+    "name": { "str": "mild steel kilij" },
+    "description": "A Turkish sabre, the kilij is quite a bit heavier than most middle eastern sabres and has a distinctive wide tip called a yelman, which gives it vicious cutting power but means it requires a special scabbard. \n\nThis one has been made of low carbon steel, and doesn't even have a proper grip or a truly sharp edge.",
```

```suggestion
    "description": "A Turkish sabre, the kilij is quite a bit heavier than most Middle Eastern sabres and has a distinctive wide tip called a yelman, which gives it vicious cutting power but means it requires a special scabbard. \n\nThis one has been made of low carbon steel, and doesn't even have a proper grip or truly sharp edge.",
```

> it requires a special scabbard

> Has SHEATH_SWORD

I'm having a big think 🤔